src/components/documentation/DocumentComponent.tsx

```diff
@@ -0,0 +1,103 @@
+// Import styled-components to define custom styles in JavaScript
+import styled from "styled-components";
+
+// ------------------- Styled Components -------------------
+
+// Wrapper: Styles the outermost container for the documentation component
+const Wrapper = styled.div`
+  border: 1px solid #121212; /* Dark border for the wrapper */
+  margin: 20px 0px; /* Vertical margin to add spacing around the component */
+`;
+
+// Container: Styles the main layout for the component's content
+const Container = styled.div`
+  padding: 20px; /* Adds inner padding around the content */
+  display: flex; /* Enables flexbox layout for content alignment */
+  justify-content: space-between; /* Distributes items with space between them */
+`;
+
+// Title: Styles the title section of the component
+const Title = styled.div`
+  display: flex; /* Enables flexbox layout */
+  justify-content: center; /* Centers the title horizontally */
+  padding-top: 20px; /* Adds space above the title */
+  font-size: 1.3rem; /* Medium font size */
```

Comment:
Only add comments when you need to explain something that is non-trivial and hard understand